src/generate-query.ts

```diff
@@ -447,10 +448,64 @@ function getNextNodefactor(variableValues: { [name: string]: any }): number {
   return 1
 }
 
+/**
+ * Returns the first slicing argument defined in the field's @listSize diretive,
```

Comment:
I think we should point to the documentation of the`@listSize` directive

Reply:
Will add a TODO until the specification / documentation of the `@listSize` directive to point to exists.